libs/Transisthor/CMakeLists.txt

```diff
@@ -1,11 +1,13 @@
+add_subdirectory(TransisthorECSLogic)
+
 set(INC_ROOT ${PROJECT_SOURCE_DIR}/libs)
 set(SRC_ROOT ${PROJECT_SOURCE_DIR}/libs/Transisthor)
 
 set(TARGET ${PROJECT_NAME}_transisthor)
 
 set(SRC
+    ${SRC_TEL}
```

Comment:
???

Reply:
It's a Child value containing all the source file of the TEL "library" (Transisthor ECS Logic)